mithril-aggregator/src/dependency_injection/builder/enablers/misc.rs

```diff
@@ -51,4 +56,31 @@ impl DependenciesBuilder {
 
         Ok(self.message_service.as_ref().cloned().unwrap())
     }
+
+    /// build an [AggregatorClient]
+    pub async fn build_master_aggregator_client(&mut self) -> Result<Arc<dyn AggregatorClient>> {
+        let master_aggregator_endpoint = &self
+            .configuration
+            .master_aggregator_endpoint
+            .clone()
```

Comment:
Maybe using `cloned` would avoid the needs to use `to_owned` bellow.

Reply:
`cloned()` is specific to iterators, but using `to_owned()` in the first place avoids the problem :+1: